haskell-tests/Generators.hs

```diff
@@ -297,6 +297,7 @@ genPayloadConfigureBaker = do
     cbTransactionFeeCommission <- liftArbitrary genAmountFraction
     cbBakingRewardCommission <- liftArbitrary genAmountFraction
     cbFinalizationRewardCommission <- liftArbitrary genAmountFraction
+    cbSuspend <- arbitrary
```

Comment:
Whether this is (possibly) present should be guarded by passing a `ProtocolVersion` or a `Bool`, since otherwise this *should* break the serialization testing for older protocol versions.

Possibly the `genInvalidPayload...` functions should also be updated.

Reply:
It's now guarded behind `protocolSupportsSuspend` (as introduced in #551). But I'm not 100% I got this right. The serialization is unchanged, as older protocols will just not set the corresponding bits in the bitmap.